Tasks/AzureNLBManagement/task.json

```diff
@@ -0,0 +1,120 @@
+        },
+        {
+            "name": "NICDetection",
+            "type": "pickList",
+            "label": "NIC Detection",
+            "required": true,
+            "options": {
+                "AutoDetectNic": "Auto select primary network interface",
+                "CustomNic": "Custom - virtual machine : network interface"
+            },
+            "defaultValue": "AutoDetectNic",
+            "helpMarkDown": "Custom: Specify mapping virtual machine and the network interface associated with the virtual machine to be added/removed from load balancer backend pool. The format to specify is: `<virtual machine1>:<network interface name1> <virtual machine2>:<network interface name2>`… and so on.\n\nAuto select:  Will select the primary network interface of the virtual machine to perform the Action of connecting/disconnecting from load balancer's backend address pool."
+        },
+        {
+            "name": "VmNicMapping",
+            "type": "multiLine",
+            "label": "VM:NIC Mapping",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "NICDetection = CustomNic",
+            "helpMarkDown": "Specify mapping virtual machine and the network interface associated with the virtual machine to be added/removed from load balancer backend pool. Specify space delimited pairs for multiple virtual machines. For example, `<virtual machine1>:<network interface name1> <virtual machine2>:<network interface name2>…` and so on."
+        }
+	],
+	"dataSourceBindings": [
+        {
+            "target": "ResourceGroupName",
+            "endpointId": "$(ConnectedServiceName)",
+            "dataSourceName": "AzureResourceGroups"
+        },
+        {
+            "target": "LoadBalancer",
+            "endpointId": "$(ConnectedServiceName)",
+            "dataSourceName": "AzureRMLoadBalancers",
+            "parameters": {
+                "ResourceGroupName": "$(ResourceGroupName)"
+            }
+        }
+    ],
+    "execution": {
+		"Node": {
+			"target": "nlbtask.js"
+		}
+	},
+    "messages": {
+        "CouldNotFetchNicDetails": "Could not fetch NIC configuration details for the Azure Virtual Machine : %s",
+        "NicDetailsFetched": "NIC configuration details fetched for the Azure Virtual Machine : %s",
+        "GettingAllNetworkInterfacesInTheResourceGroup": "Getting all network interfaces in the Resource Group : %s",
+        "RequestingForAuthToken": "Requesting for authorization token : %s",
+        "ConnectingVMtoLB": "Connecting the Azure Virtual Machine to the Load Balancer - %s's backend pool.",
+        "DisconnectingVMfromLB": "Disconnect the Azure Virtual Machine from the Load Balancer - %s's backend pool.",
+        "setNICStatus": "Network interface %s set successfully",
+        "MaxRetriesExceededForPut": "Maximum retries exceeded for making the PUT request for the Network Interface : %s",
```

Comment:
Don't need to call out PUT. Just let user know that retries to set network interface properties failed.